regparser/tree/xml_parser/reg_text.py

```diff
@@ -292,7 +293,9 @@ class RegtextParagraphProcessor(paragraph_processor.ParagraphProcessor):
     MATCHERS = [paragraph_processor.StarsMatcher(),
                 paragraph_processor.TableMatcher(),
                 paragraph_processor.FencedMatcher(),
-                extract_processor.ExtractMatcher(),
+                flatsubtree_processor.FlatsubtreeMatcher(tags=['EXTRACT'],
+                                                         node_type="extract"),
```

Comment:
Similar to the above, would it make more sense to use the constant, `Node.EXTRACT`?